utils/swift_build_support/tests/products/test_ninja.py

```diff
@@ -114,7 +120,7 @@ def test_build(self):
 + {expect_env}{python} configure.py --bootstrap
 + popd
 """.format(
-            source_dir=os.path.join(self.workspace.source_root, 'ninja'),
-            build_dir=os.path.join(self.workspace.build_root, 'ninja-build'),
+            source_dir=self.workspace.source_dir('ninja'),
```

Comment:
Does this formatting pass `flake8`? Seems like the indentation here would cause an issue with linters. Usually they only want a single indentation level in these cases.

Reply:
I think it did, but I’m not sure why. I will see if moving things around avoids the extra line.

Reply:
So `flake8` did like this indentation. I think the reason is because the text between `"""` is unindented to the first column to match the output that the test is comparing against. The indentation of the parameters was one level more than the original indentation before the `"""` block. I have moved the first line up (so the parentheses and the first argument are in the same line), and it doesn’t look as strange as before (and still passes `flake8`).